ledger/accountdb.go

```diff
@@ -2693,7 +2693,7 @@ func (qs *accountsDbQueries) lookupKeyValue(key string) (pv persistedKVData, err
 // The DB comparison for blob typed primary key is lexicographic, i.e., byte by byte.
 // In this way, we can introduce an interval that a primary key should be >= some prefix, < some prefix increment.
 // A corner case to consider is that, the prefix has last byte 0xFF, or the prefix is full of 0xFF.
-// The first case can be solved by carrying, e.g., prefix = 0x1EFF -> 0x1F00
+// The first case can be solved by carrying, e.g., prefix = 0x1EFF -> interval being >= 0x1EFF and < 0x1F
```

Comment:
Leaving the 00's on or taking them off is the same.

Reply:
maybe a subtle difference: consider upper limit `0x1F00` and `0x1F`, and lower bound `0x1EFF`. Is `0x1F` inside `0x1EFF` and `0x1F00`? If length is also considered, then it should not be inside the interval, is that correct?